odoo/tools/mail.py

```diff
@@ -315,7 +315,7 @@ def is_html_empty(html_content):
     """
     if not html_content:
         return True
-    tag_re = re.compile(r'\<\s*\/?(?:p|div|span|br|b|i|font)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
+    tag_re = re.compile(r'\<\s*\/?(?:p|div|section|span|br|b|i|font)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
```

Comment:
Is there by any chance a small test somewhere we can modify to add this use case? Just to make sure it works as expected.

Reply:
@awa-odoo 
Yes, I added a case with a mishmash of some of the ones that aren't really tested